### PR TITLE
Update to version 3.7.1

### PR DESCRIPTION
**Pull request by rth**
Continues https://github.com/conda-forge/gensim-feedstock/pull/23

Closes https://github.com/conda-forge/gensim-feedstock/pull/23

### CODE REVIEW

File: recipe/meta.yaml [hunk 1]
```diff
@@ -22,7 +22,7 @@ requirements:
   host:
     - python
     - pip
-    - numpy
+    - numpy 1.11.3
```
Comment (rth) [inline recipe/meta.yaml]: Otherwise an older unsupported numpy 1.9 is used

**Comment by mpenkov** [comment]
@rth Hi, I'm the new maintainer of gensim (and smart_open). I'm trying to restart the failed appveyor build, but lack permissions for the gensim-feedstock and smart_open-feedstock repos. Would you be able to give them to me? If not, who's the most appropriate person to ask?

Also, please remove all previous maintainers from the feedstocks. @menshikh-iv has moved on and no longer maintains the project, as have the other maintainers before him.

Pinging @piskvorky (the owner of gensim and smart_open) just in case.

**Comment by souravsingh** [comment]
@mpenkov I should be able to add you as a maintainer for the feedstock.

**Comment by souravsingh** [comment]
@conda-forge-admin, please update team

**Comment by mpenkov** [comment]
@bgruening @ocefpaf Ping. Could you please have a look at this, or suggest a more appropriate person to ask? Our gensim and smart_open feedstocks are starting to fall behind the most recent versions because I don't have the appropriate permissions.

**Comment by mpenkov** [comment]
@souravsingh If possible, please add me to gensim and to smart_open.

**Comment by ocefpaf** [comment]
> @bgruening @ocefpaf Ping. Could you please have a look at this, or suggest a more appropriate person to ask? Our gensim and smart_open feedstocks are starting to fall behind the most recent versions because I don't have the appropriate permissions.

@mpenkov the maintainers here are @souravsingh and @menshikh-iv, they are the appropriated people to ask. BTW, you can send a PR adding yourself and help them maintain this feedstock.

@souravsingh [invoking the bot](https://github.com/conda-forge/gensim-feedstock/pull/25#issuecomment-481311008) without adding the maintainer to the list in the `meta.yaml` won't help.

**Comment by piskvorky** [comment]
@ocefpaf, @souravsingh and @menshikh-iv are no longer associated with Gensim. The maintainers are me (@piskvorky) and @mpenkov . Can you update the maintainer/contributor/owner list please?

**Comment by souravsingh** [comment]
@mpenkov should receive an invite to join the feedstock repo.

**Comment by ocefpaf** [comment]
> @ocefpaf: @souravsingh and @menshikh-iv are no longer directly associated with Gensim. The maintainers are me (@piskvorky) and @mpenkov.
> 
> Can you update the maintainer/contributor/owner list here please? Cheers.

The only way to do that is via a PR like @souravsingh did in https://github.com/conda-forge/gensim-feedstock/pull/27

**Pull request merged**